AWSAuthSDK/Sources/AWSMobileClient/AWSMobileOptions.swift

```diff
@@ -14,18 +14,22 @@ import Foundation
     @objc public let logoImage: UIImage?
     /// The background color of the sign-in screen.
     @objc public let backgroundColor: UIColor?
-    
+    /// The background color applited to the bottom panel of the sign-in screen.
+    @objc public let backgroundBottomColor: UIColor?
+    /// The view tint (foreground) color used for highlighted elements (button background, links).
+    @objc public let tintColor: UIColor?
     
     /// Initializer for the drop-in UI configuration.
-    ///
-    /// - Parameters:
```

Comment:
I'm not saying they're *great* comments, but they still give context to the options. Why remove them rather than retaining the existing and adding new ones for the new properties, and making their use cases clearer?

Reply:
I removed this particular comment because the properties themselves already have comments, so I considered these redundant. But no strong opinion on this, I can put it back if you believe they're valuable.